src/main.ts

```diff
@@ -35,9 +35,25 @@ Vue.use(VueCodemirror);
 // Production
 Vue.config.productionTip = false;
 
-new Vue({
-  router,
-  store,
-  vuetify,
-  render: h => h(App),
-}).$mount('#app');
+/**
+ * initialize app.
+ */
+const initApp = () => {
+  new Vue({
+    router,
+    store,
+    vuetify,
+    render: h => h(App),
+  }).$mount('#app');
+};
+
+// Load data for global config.
+fetch(process.env.BASE_URL + 'config.json')
+  .then(response => response.json())
+  .then(config => {
+    Vue.prototype.$config = config;
+    initApp();
+  })
+  .catch(() => {
+    initApp();
+  });
```

Comment:
How about the following:
```suggestion
  .then(config => {
    Vue.prototype.$config = config;
  })
  .finally(() => {
    initApp();
  });
```
Do you think we would need the `catch`? I tested it shortly and it looked as if it would work correctly even without it.